recipes/einsteinpy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "einsteinpy" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 8a8815ae9341f4b544c496a697342892
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True
+
+requirements:
+  build:
+    - python
```

Comment:
For this recipe `python` and `setuptools` should be in the `host` dependencies.

Reply:
@chrisburr Okay, I have made the changes.